test/src/main/configurations/MainConfig/XmlValidator/GetData/XSD_Simple_Types_c0.8.xsd

```diff
@@ -1 +1 @@
			<xsd:maxLength value="255"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="accountAttributeValue">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="255"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="accountAttributeValueDescription">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="255"/>
		</xsd:restriction>
	</xsd:simpleType>
	<!--*********************************BIK ContactPointUsage Simple Types*************************************************-->
	<xsd:simpleType name="preferenceRank">
		<xsd:restriction base="xsd:positiveInteger">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="preferredUsage">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="3"/>
		</xsd:restriction>
	</xsd:simpleType>
	<!--*********************************BIK PartyInternalAgreementRole Simple Types***************************************-->
	<xsd:simpleType name="partyInternalAgreementRoleTypeName">
		<xsd:restriction base="xsd:positiveInteger">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="partyInternalAgreementRoleStatus">
		<xsd:restriction base="xsd:positiveInteger">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="partyInternalAgreementRoleContactStatus">
		<xsd:restriction base="xsd:positiveInteger">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
	<!--*********************************BIK Qualifications Simple Types********************************************************-->
	<xsd:simpleType name="qualificationGroup">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="40"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="qualificationsName">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="40"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="proficiency">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="40"/>
		</xsd:restriction>
	</xsd:simpleType>
	<!--*********************************BIK Responsible Simple Types********************************************************-->
	<xsd:simpleType name="personalNumber">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="10"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="assignmentType">
		<xsd:restriction base="xsd:string">
			<xsd:maxLength value="1"/>
		</xsd:restriction>
	</xsd:simpleType>
	<!--*********************************BIK RelationshipAttributes Simple Types********************************************************-->
	<xsd:simpleType name="relationshipattributes_id">
		<xsd:restriction base="xsd:integer">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
	<xsd:simpleType name="relationshipattributes_value">
		<xsd:restriction base="xsd:integer">
			<xsd:maxInclusive value="99"/>
		</xsd:restriction>
	</xsd:simpleType>
</xsd:schema>

```

Comment:
Kan je er voor zorgen dat dit bestand Unix line endings krijgt (LF)? Nu zijn het Macintosh line endings (CR), en is niet te zien wat er veranderd is.

Reply:
Gedaan